### PR TITLE
Remove all critical/high security warnings

### PR DESCRIPTION
**Pull request by jlamarr22**
## Change Description
Remove all critical/high security issues

Before:
Screen Shot 2019-12-12 at 10 13 43 AM

After:
Screen Shot 2019-12-12 at 1 26 32 PM

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

**Review by jeffreyssmith2nd — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -59,8 +59,8 @@
     "argparse": "1.0.10",
```
Comment (GreenBusDriver) [inline package.json]: Please lock the versions in the "dependencies" section.

**Review by GreenBusDriver — APPROVED** [review]
Just version lock the "dependencies", then you're good to go

**Comment by jeffreyssmith2nd** [comment]
I generally recommend making a separate commit for any documentation changes. That will make it easier for reviewers to find actual code changes.

**Review by jeffreyssmith2nd — APPROVED** [review]

**Review by GreenBusDriver — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -30,39 +30,34 @@
...
+    "yargs": "15.0.2"
   },
   "resolutions": {
     "argparse": "1.0.10",
-    "underscore.string": "3.3.5",
-    "merge": "1.2.1",
-    "handlebars": "4.1.2",
-    "lodash": "4.17.11",
-    "js-yaml": "3.13.1",
-    "marked": "^0.6.2"
```
Comment (GreenBusDriver) [inline package.json:65]: Yay!  So many fewer "resolutions"!!!  Thanks for doing this!

**Review by GreenBusDriver — APPROVED** [review]

**Pull request merged**